scale-tests/setup_streaming.py

```diff
@@ -58,6 +58,19 @@
 SUPPORTED_MULTIPLE_CLUSTER_SERVICES = ['kafka', 'confluent-kafka', 'beta-kafka']
 
 
+def setup_security(service_name: str) -> typing.Dict:
+    """
+    Adds a service account an secret for the specified service name.
```

Comment:
(sp) "*and* secret"

Reply:
@susanxhuynh this is addressed in #341. I think some of the edits ended up in this branch.